packages/sdk-react/src/ui/DueDateCell/DueDateCell.tsx

```diff
@@ -37,9 +37,10 @@ export const DueDateCell = ({ data }: DueDateCellProps) => {
           fontWeight="bold"
           fontSize="small"
         >
-          {t(i18n)`Overdue by ${overdueDays} ${
-            overdueDays === 1 ? 'day' : 'days'
-          }`}
+          {overdueDays > 0 &&
```

Comment:
There is a function called `plural` in Lingui. Check the example in the file `packages/sdk-react/src/components/receivables/InvoiceDetails/ExistingInvoiceDetails/components/reminderCardTermsHelpers.tsx`
Let's use a single approach to pluralize translations